tests/neo4j/datatypes.py

```diff
@@ -104,6 +104,30 @@ def testShouldEchoNestedLists(self):
         self.createDriverAndSession()
         self.verifyCanEcho(types.CypherList(test_lists))
 
+    def testShouldEchoListOfMaps(self):
+        test_list = [
+            types.CypherMap({
+                "a": types.CypherInt(1),
+                "b": types.CypherInt(2)
+            }),
+            types.CypherMap({
+                "c": types.CypherInt(3),
+                "d": types.CypherInt(4)
+            })
+        ]
+
+        self.createDriverAndSession()
+        self.verifyCanEcho(types.CypherList(test_list))
+
+    def testShouldEchoMapOfLists(self):
+        test_map = {
+            'a': types.CypherList([types.CypherInt(1)]),
+            'b': types.CypherList([types.CypherInt(2)])
+        }
+
+        self.createDriverAndSession()
+        self.verifyCanEcho(types.CypherMap(test_map))
```

Comment:
Please use snake case (`test_should_echo_list_of_maps` instead of `testShouldEchoListOfMaps`, etc.) for variables, functions, and methods.

Reply:
The files was all in camel case. Should I put everything in snake_case?

Reply:
Preferably yes. I would at least not add more code the ignores the style guide.

Reply:
So I will change the name of all the tests in this file.